### PR TITLE
Introduction is improved & usage tips added

### PR DESCRIPTION
**Pull request by PriyanshuSahani**
I have edited the introduction to CCExtractor .Text under the headder 'Google Code-in 2017' was not much appealing ,so I put those paragraphs as points marked by bullets . Under 'Installation and usage', I have added  a line . I have added paragraphs namely ,'How easy is it to use CCExtractor ?' & 'How to use subtitles once they are in a separate file?'.

Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [ ] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that another pull request for this purpose does not exist.
- [ ] I have considered, and confirmed that this submission will be valuable to others.
- [ ] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [ ] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---

{pull request content here}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
 ![logo](https://avatars3.githubusercontent.com/u/7253637?v=3&s=100)
  
-# CCExtractor 
+# CCExtractor
+What's CCExtractor?
 
+A tool that analyzes video files and produces independent subtitle files from the closed captions data. CCExtractor is portable, small, and very fast. It works in Linux, Windows, and OSX. 
 [![Build Status](https://travis-ci.org/CCExtractor/ccextractor.svg?branch=master)](https://travis-ci.org/CCExtractor/ccextractor)
 
-CCExtractor is a tool that produces subtitles from TV use. Global accessibility (all users, all content, all countries) is the goal. With so many different formats, this is a constantly moving target, but we intend to keep up with all sources and formats.
```
Comment (xvkdev) [inline README.md]: Why was this removed?

Comment (cfsmp3) [inline README.md]: What's wrong with this?

**Comment by PriyanshuSahani** [comment]
I removed the introduction that was originally given by CCExtractor coz
according to me the first paragraph should just give the idea of project
and may no tell that the aim of CCExtractor is 'Global Accessibility'.This
term can be used later in the README .I may be wrong in it ,so I have re
edited the README.MD file and have retained the previous content and sent
the link to PR
Kindly check
Thank You
